config/index.js

```diff
@@ -53,6 +54,10 @@ const config = {
     },
     regex: {
       woloxEmail: process.env.WOLOX_EMAIL_REGEX
+    },
+    pagination: {
+      limit: parseInt(process.env.LIMIT_PAGINATION) || 25,
+      offset: parseInt(process.env.OFFSET_PAGINATION) || 1
```

Comment:
Why offset 1?

Reply:
It should be 0, what do you think?

Reply:
I Agree. Also in other companies it was requested to define it from 1. Solved!